devex/cross-nuget/README.md

```diff
@@ -0,0 +1,136 @@
+# Cross-Platform Open Enclave SDK NuGet Package
+
+This directory contains the scripts used to build a cross-platform NuGet package
+for the Open Enclave SDK. The resulting package may be used on Windows to build
+SGX enclaves as well as on Linux to build SGX and OP-TEE enclaves for the Scalys
+Grapeboard (LS-1012A) and QEMU ARMv8.
+
+The SGX builds are suitable both for SGX1 systems as well as for systems that
+support FLC. All SGX builds contain enclave libraries with and without LVI
+mitigations.
+
+## Environment Setup
+
+Two build systems are required: the first must run Windows Server 2019 with GUI
+and the second must run Ubuntu 18.04 LTS or later (Desktop and Server are both
+OK). These two environments may exist on physical systems or in virtual
+machines, it does not matter.
+
+This section explains how to set up both environments.
+
+### Windows Server 2019
+
+1. Install Windows Server 2019 with GUI.
+2. Download the SDK's [Windows prerequisites installation
+   script](https://github.com/openenclave/openenclave/blob/master/scripts/install-windows-prereqs.ps1).
+3. If present, comment out the line in the script that starts with:
+   ```powershell
+   New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Services\sgx_lc_msr\Parameters" [...]
```

Comment:
Why is this needed? t.

Reply:
I double checked, and it isn't. Removed.